### PR TITLE
Stephen Drollinger Bio

### PR DESCRIPTION
**Pull request by Valimere**
Initial bio for Stephen Drollinger

### CODE REVIEW

File: src/data/team-data.js [hunk 1]
```diff
@@ -33,7 +33,8 @@ export default [
   },
   {
     title: 'Stephen Drollinger',
-    summary: 'Stephen Drollinger is ...',
-    content: 'Stephen Drollinger is ...',
+    github: 'https://github.com/Valimere',
+    summary: 'Stephen Drollinger is a software engineer at Ironnet Cybersecurity working on the Detection Development turning new analytics into production code. ',
```
Comment (jmymy) [inline src/data/team-data.js]: Stephen Drollinger is a software engineer at Ironnet Cybersecurity, working on Detection Development, turning new analytics into production code.

File: src/data/team-data.js [hunk 2]
```diff
@@ -33,7 +33,8 @@ export default [
...
+    content: 'Stephen Drollinger started at Ironnet Cybersecurity in a DevOps role supporting a product made up of many microservices. Responsibilities included implementing and maintaining physical hardware, Openstack, customer deployments, CI/CD and more. He transitioned to a software engineer on completion of his degree in computer science. Now he works on the Detection Development team turning new analytics into production code. Stephen also has a Network Engineering background and recently completed the AWS certified developer associate',
```
Comment (jmymy) [inline src/data/team-data.js]: > Stephen Drollinger is a software engineer at Ironnet Cybersecurity. He has held DevOps roles supporting products, implemented and maintained physical hardware, Openstack, customer deployments, CI/CD, and more. He recently transitioned to a software engineer, working on the Detection Development team, turning new analytics into production code.

Thoughts? @WTPascoe youre better at this than I

Comment (WTPascoe) [reply src/data/team-data.js]: How about this:

> Stephen Drollinger is a software engineer at IronNet Cybersecurity. He has held a number of DevOps roles, implemented and been responsible for physical hardware, Openstack, customer product deployments, CI/CD, and more. He recently transitioned to a software engineering role working on the Detection Development team, turning new analytics into production code.

File: src/data/team-data.js (hunk 1 above)
Comment (WTPascoe) [reply src/data/team-data.js]: > Stephen Drollinger is a software engineer with a strong DevOps background. He currently works at IronNet Cybersecurity on Detection Development, turning new analytics into production code.

**Pull request merged**